scl/paloalto/panos.conf

```diff
@@ -0,0 +1,97 @@
+#############################################################################
+# Copyright (c) 2020 Balabit
+# created by mileek@gmail.com
+# This program is free software; you can redistribute it and/or modify it
```

Comment:
The copyright check is failing here. Could you please double check it? You can try to copy from another file, and insert your name similarly as here: https://github.com/syslog-ng/syslog-ng/blob/06df1d712cf0608845eba0662e2cba7f85703af8/scl/graphite/plugin.conf#L3

The copyright script is very strict, all whitespaces, enters matter. If you get stuck, let me know, and I try to find out what went wrong.

Reply:
OK